### PR TITLE
Update mission settings in right place

### PR DESCRIPTION
**Pull request by hamishwillee**
This puts changes from release 3.2 into body of the Planview doc

### CODE REVIEW

File: en/PlanView/PlanView.md [hunk 1]
```diff
@@ -108,22 +115,76 @@ The list of commands displayed in the dialog can be filtered by category. For ex
...
+
+#### Camera
+
+The camera section allows you to specify a camera action to take, control the gimbal and set your camera into photo or video mode.
+
+![MissionSettings Camera Section](../../assets/plan/MissionSettingsCameraSection.jpg)
+
+The available camera actions  are:
+
+*  Continue current action
```
Comment (hamishwillee) [inline en/PlanView/PlanView.md]: @DonLakeFlyer This didn't appear to be an option. Is it conditional on something? (this comes from release notes).

Comment (DonLakeFlyer) [reply en/PlanView/PlanView.md]: Should be: "No change (continue current action)" or something like that.

Comment (DonLakeFlyer) [reply en/PlanView/PlanView.md]: The combo box item ius "No change"

Comment (hamishwillee) [reply en/PlanView/PlanView.md]: Cheers. Fixed in https://github.com/mavlink/qgc-user-guide/commit/350738f7e1583aeadc5a0e0f0477649bde4428fb

**Pull request merged**